hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/checksum/ContainerMerkleTreeTestUtils.java

```diff
@@ -87,8 +91,8 @@ public static void assertTreesSortedAndMatch(ContainerProtos.ContainerMerkleTree
    *     "bytesPerChecksum" amount of data and are assumed to be contiguous.
    * @return The ChunkInfo proto object built from this information.
    */
-  public static ChunkInfo buildChunk(ConfigurationSource config, int indexInBlock, ByteBuffer... chunkChecksums)
-      throws IOException {
+  public static ContainerProtos.ChunkInfo buildChunk(ConfigurationSource config, int indexInBlock,
+                                                     ByteBuffer... chunkChecksums) {
```

Comment:
nit. Better to indent this and the other change 4 spaces below the method like others in this class. Technically checkstyle doesn't complain though.